web/gui/dashboard_info.js

```diff
@@ -794,6 +794,31 @@ netdataDashboard.submenu = {
     'ntpd.peers': {
         title: 'peers',
         info: 'Statistics of the peer variables for each peer configured in <code>/etc/ntp.conf</code> as shown by the readvar billboard <code>ntpq -c "rv &lt;association&gt;"</code>, while each peer is assigned a nonzero association ID as shown by <code>ntpq -c "apeers"</code>. The module periodically scans for new/changed peers (default: every 60s). <b>ntpd</b> selects the best possible peer from the available peers to synchronize the clock. A minimum of at least 3 peers is required to properly identify the best possible peer.'
+    },
+
+    'filesystem.EXT4_latency': {
+        title: 'EXT4 Latency',
+        info: 'Latency is the time it takes for an event to be completed. We calculate the difference between the calling and return times, we get the logarithmic for the final result and we sum one value to the respective bin. Based on the eBPF <a href="http://www.brendangregg.com/blog/2016-10-06/linux-bcc-ext4dist-ext4slower.html" target="_blank">ext4dist</a> from BCC tools.'
```

Comment:
I see that the link _contains_ latency description. In case of ext4 it is

> This "latency" is measured from when the operation was issued from the VFS interface to the file system, to when it is completed. This spans everything: block device I/O (disk I/O), file system CPU cycles, file system locks, run queue latency, etc. 

Do we get the same numbers as `ext4dist` provides?

Reply:
My point is that _difference between the calling and return times_ sounds like implementation details and not latency description (in the general sense). I could be wrong, not sure 🤷‍♂️

Reply:
If our goal is to provide more a description of latency I agree with what @ilyam8 mentions and we should write something closer to what is on the link for latency description. If the goal is to provide the link and add to that how we calculate it seem ok.

From my pov, a person who doesn't know much about these tech details, it would be more helpful having something more close to a latency description, but not sure that is the intended goal.

Reply:
> Do we get the same numbers as ext4dist provides?

@ilyam8  we do not show the exact same numbers of `ext4dist`, because we are using `INCREMENTAL` algorithm, but we collect like them.

I will bring more information for our dashboard related to latency in the next commit.

Reply:
@ilyam8  and @hugovalente-pm I did not copy and paste texts, because I think this is not our goal, and probably this could create problems, so I adjusted the text that we are showing on our dashboard, please, take a look when you have a time.